maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/StartupReportConfiguration.java

```diff
@@ -107,8 +119,11 @@ public StartupReportConfiguration( boolean useFile, boolean printSummary, String
         this.rerunFailingTestsCount = rerunFailingTestsCount;
         this.xsdSchemaLocation = xsdSchemaLocation;
         String charset = trimToNull( encoding );
-        this.encoding = charset == null ? Charset.defaultCharset() : Charset.forName( charset );
+        this.encoding = charset == null ? UTF_8 : Charset.forName( charset );
```

Comment:
is this a behaviour change ? should we document it ?

Reply:
We have to find the Jira issue because this is what the users wanted. They expected wider charset or the same as it was in forked JVM. The problem is that two JVMs (Maven and forked) may have two different charsets. It was a problem that 8 bit charsets are not compatible. Due to we transfer all strings over the pipes via UTF-8 (SUREFIRE-1222), we should not use shorter charset for `charset = null`. This is one point where both SUREFIRE-1222 and SUREFIRE1546 are touched.

Reply:
We can separate this change line to a new commit for SUREFIRE-1222. WDYT?